endpoints/cookie_sync.go

```diff
@@ -125,6 +132,21 @@ func (c *cookieSyncEndpoint) parseRequest(r *http.Request) (usersync.Request, pr
 		}
 	}
 
+	if request.Account != "" {
+		accountInfo, errs := accountService.GetAccount(context.Background(), c.config, c.accountsFetcher, request.Account)
+		if len(errs) > 0 {
+			return usersync.Request{}, privacy.Policies{}, errs[0]
```

Comment:
Yeah I was also confused about this, because account service returns an array of errors, but the parse request function only returns a single error. Veronika, are you suggesting we refactor the GetAccount function?